docs/kola.md

```diff
@@ -66,6 +66,9 @@ Example format of the file:
     # If no streams are specified, test will be skipped on all streams
     - stream1
     - stream2
+  # The test will only be skipped until this date (will resume on the date)
+  # Format: YYYYMMDD
```

Comment:
Minor: if it's equally easy to support, I find `YYYY-MM-DD` as a format more legible. Though obviously fine with this too if others prefer.

Reply:
I'm cool either way

Reply:
I'm good for both, but I do agree that your suggestion is more legible. The original format is just the format that was used as an example in the linked issue. I'm not really bound to it.